lib/DataSet.js

```diff
@@ -304,23 +304,23 @@ DataSet.prototype.update = function (data, senderId) {
  *     get()
  *     get(options: Object)
  *
- *     get(id: Number | String)
- *     get(id: Number | String, options: Object)
+ *     get(id: number | string)
+ *     get(id: number | string, options: Object)
  *
- *     get(ids: Number[] | String[])
- *     get(ids: Number[] | String[], options: Object)
+ *     get(ids: number[] | string[])
```

Comment:
Wrt. previous comment, this looks OK to me, although perhaps too strict. I can imagine strings and numbers mixed in the same array.

Perhaps we should just forbid that. Or make a preemptive type-conversion.

Reply:
Silly me; it's not even `@param` tags. Sorry.